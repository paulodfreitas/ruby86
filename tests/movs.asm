.module entry_code
.pseg
.org 0
        irmovl $33, %eax
        irmovl $3000, %ebx
        rrmovl %eax, %ecx
        rmmovl %eax, 3(%ebx)
        mrmovl 3(%ebx), %edx
        halt
.end
