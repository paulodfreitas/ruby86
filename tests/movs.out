@ 0000
30
80
33
00
00
00
30
83
00
30
00
00
20
01
40
03
03
00
00
00
50
23
03
00
00
00
10
@ 001b
